scripts/rollup/results.json

```diff
@@ -4,8 +4,8 @@
       "filename": "react.development.js",
       "bundleType": "UMD_DEV",
       "packageName": "react",
-      "size": 58569,
-      "gzip": 16268
+      "size": 59151,
```

Comment:
Please revert changes to this file

Reply:
Will do. It was not clear for mr whether it should be committed or not, so I decided to make sure every commit that changes the build has this file on par. Is there any note about this file in the contribution guidelines? I didn't find.

Reply:
Not really. I think at some point we wanted to delete it completely since we now track builds on PRs instead.

Reply:
I think it's used for local builds only. I don't mind removing it altogether in a separate PR given that it doesn't break the build command.

Reply:
Done during rebase to latest `fasebook/master`.